expression/builtin_encryption_vec.go

```diff
@@ -135,11 +136,35 @@ func (b *builtinRandomBytesSig) vecEvalString(input *chunk.Chunk, result *chunk.
 }
 
 func (b *builtinMD5Sig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinMD5Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+	result.ReserveString(n)
+	digest := md5.New()
+	for i := 0; i < n; i++ {
+		digest.Reset()
```

Comment:
Sorry it's my bad,
we could put `digest.Reset` after `digest.Sum`, so if we've continued before calculating, we need not reset.

Reply:
I will fix it.